pkgs/applications/audio/ardour/ffmpeg_harvid.nix

```
@@ -0,0 +1,26 @@
+{ stdenv, ffmpeg }:
+
+stdenv.mkDerivation {
+  name = "ffmpeg_harvid";
+
+  buildInputs = [ ffmpeg ];
+
+  phases = [ "installPhase" ];
+
+  installPhase = ''
```

```suggestion
  buildCommand = ''
```
And don't set `phases`.